modules/stanford_jumpstart_vpsa_permissions/stanford_jumpstart_vpsa_permissions.install

```diff
@@ -216,5 +216,10 @@ function stanford_jumpstart_vpsa_permissions_update_7401_get_permissions() {
     'module' => 'node',
   );
 
-  return $permissions;
+  $permissions_modules = user_permission_get_modules();
+  // If a permission doesn't exist, filter it out to prevent errors when trying
+  // to add a non-existent permission.
+  return array_filter($permissions, function ($permission) use ($permissions_modules) {
```

Comment:
Am I correct that this is an anonymous function? `diseval` might puke on it if so.

Reply:
Theres several modules that do a similar thing. including Guzzle in capx. Hopefully diseval doesn't hate it https://github.com/SU-SWS/stanford_capx/blob/7.x-3.x/includes/vendor/guzzlehttp/psr7/src/Uri.php#L634

Reply:
and its used alot in D8.

Reply:
FWIK Anon functions are ok. `create_function` is not.

https://github.com/mk-j/PHP_diseval_extension

Reply:
Ah that's right. My memory was fuzzy.